npm/ls-lint/postinstall.js

```diff
@@ -0,0 +1,24 @@
+const { spawn } = require("child_process");
+const { join } = require("path");
+
+const isCI = process.env.CI;
+
+if (!isCI) {
+  if (process.platform === 'darwin') {
```

Comment:
please use switch case here, cause there could be some multi cases like here:
 
https://github.com/electron/electron/blob/master/npm/install.js#L67